src/rules/hindi.toml

```diff
@@ -0,0 +1,19 @@
+min_trimmed_length = 0
+min_word_count = 0
+max_word_count = 50
```

Comment:
how long does it take to speak 50 words out aloud? We're aiming for something around 7 seconds max (if I remember correctly?).

Reply:
Hmm, then 50 words would be too long. 25 words max should be a more reasonable limit.